tests/test_message.py

```diff
@@ -29,6 +29,17 @@ class Pet(Message):
 
         self.assertEqual("Foo", from_object(Pet, {"name": "Foo"}).name)
 
+    def test_message_reserved_word(self):
```

Comment:
Shouldn't create a unittest for undesirable behavior. Remove test and create an issue instead.